templates/kafka-proxy-svc.yaml

```diff
@@ -31,8 +31,8 @@ spec:
   type: ExternalName
   # Specify the FQDN of the service to use. Pod selectors can't select outside of namespace
   # Docs suggest FQDN can be shortened inside current namespace, and inside current cluster, but this doesn't work in AWS
-  {{- if eq .Values.global.hosting "aws" }}
+  {{- if .Values.global.kafka.install }}
```

Comment:
wouldnt it makes sense to do an `if you deployed kafka and redis, and the hosting is aws`? 
Doesnt this affect the local deploy?